src/main/resources/view/InternApplicationListPanel.fxml

```diff
@@ -3,6 +3,6 @@
 <?import javafx.scene.control.ListView?>
 <?import javafx.scene.layout.VBox?>
 
-<VBox xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
-  <ListView fx:id="internApplicationListView" VBox.vgrow="ALWAYS" />
+<VBox fx:id="internApplicationVbox" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
```

Comment:
I think you can remove the `fx:id="internApplicationVbox"` here, it doesn't have a reference anywhere in the code.